lib/utils.js

```diff
@@ -36,6 +36,9 @@ function normalizeType(params) {
   type.overrides = _.map(params.overrides || params.override || {}, (overrideDef, name) => {
     const override = normalizeType(overrideDef);
     override.name = name;
+    if (overrideDef.until && !(overrideDef.until instanceof Date)) {
```

Comment:
If you are expecting a timestamp I'd validate isNaN and reject everything else because Date behavior with other kind of things (as string) is pretty complicate, unexpected and changing between node versions.

Reply:
Can be string too.

Reply:
Changing between node versions? `Date` class should be stable.

Reply:
the string parsing depends on the timezone set on the machine unless ISO with locale is used; that's pretty unexpected IMHO you don't usually know the timezone a server use. Also regarding string parsing changing between version check the behavior on node 0.3.x (it think it was that)vs current... Date for strings has always been polemic, even in the browser land different browsers used to implement it in different ways, now it has been standarized but IMO it is more problematic than it worth; if you want to parse string I'd specify the format explicitly.